jxp/scripts/ch13ex3.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Person;
+use v5.10;
+
+say "Without parameter";
+Person->speak();
+
+say "With parameter";
+Person->speak("Hello World!");
```

Comment:
Would have been good to include an animal trying to say hello to prove that the error is thrown correctly